### ISSUE TITLE
PHP Error after allocating flow to the process "Heating Water".

### ISSUE THREAD

**Issue by Xandalfius** [Ecoxl/celero#72]
https://32038.hostserv.eu/cpscoping/42094/3451/allocation
I allocated the flow: "Heat air water heat Pump CH (output)" to the Process "Heating Water", the inputs can be seen on the first screenshot.

![Error_Allocation_input](https://user-images.githubusercontent.com/69843619/90647979-4234ef00-e239-11ea-8212-4a866336ac44.PNG)

The following Error occured:
<img width="935" alt="Error_Allocation_heating water+heat" src="https://user-images.githubusercontent.com/69843619/90648210-8a541180-e239-11ea-95bd-46979cf7dd79.PNG">

I reproduced it several times, the issue seems to be the Process "Heating Water" as I was able to allocate the flow to other Processes, but no other Flows to the Process Heating Water.
Interestingly, nothing shows on the "allocation"-page in the Dataset Manager or under "View CP Potentials Identifications", but it appears as empty Lines in the "KPI View and Edit" Table:
<img width="948" alt="Error_Allocation_KPI" src="https://user-images.githubusercontent.com/69843619/90648585-02223c00-e23a-11ea-8297-a9ef7f6c316b.PNG">
Also there is no way to delete it from there. Even deleting the Process "Heating Water" did not remove the empty lines.

**Comment by Xandalfius** [comment]
Also probably because of this Bug, the Dropdown-menu on the CBA-Page doesn't seem to work.
![image](https://user-images.githubusercontent.com/69843619/91054980-4fc8eb00-e624-11ea-8a6e-3047f56e15e6.png)

**Comment by tcgumus** [comment]
Yes, i think it is because "Heating Water" process is an old process that need to be removed in db and added again in UI. 
@Ocimum-basilicum can you help with that? I think we need to delete 303 id from t_prcss and related tables. Or maybe we need to check t_prcss table for 303 id and add it manually.
